chain/network/src/peer_manager/connection/mod.rs

```diff
@@ -25,6 +27,27 @@ use std::sync::{Arc, Weak};
 #[cfg(test)]
 mod tests;
 
+impl tcp::Tier {
+    pub(crate) fn is_allowed(self, msg: &PeerMessage) -> bool {
```

Comment:
add more context on why we allow only these types of messages on tier1.

Reply:
done